backend/infrahub/core/initialization.py

```diff
@@ -264,7 +265,7 @@ async def create_branch(
     description = description or f"Branch {branch_name}"
     branch = Branch(
         name=branch_name,
-        status="OPEN",
+        status=BranchStatus.ACTIVE.value,
```

Comment:
Highlighted as per the discussions above, here we change the status from "OPEN" to "ACTIVE" but only on new installations.